packages/frontend/src/components/project/ContractsUpdated.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+
+import { cn } from '../../utils/cn'
+
+export function ContractsUpdated() {
```

Comment:
It should be more prominent, RED shouting colors at you instead of peaceful blue